docs/pages/home.js

```diff
@@ -0,0 +1,101 @@
+// @flow strict
+import { Box, Button, Flex, Heading, Text } from 'gestalt';
+import { type Node } from 'react';
+import HomePageCard from '../components/HomePageCard.js';
+import HomePageSection from '../components/HomePageSection.js';
+import Page from '../components/Page.js';
+import {
+  HowToWork,
+  Development,
+  Design,
+  A11Y,
+  Color,
+  DesignTokens,
+  Hero,
+} from '../graphics/HomePageIcons.js';
+
+export default function HomePage(): Node {
+  return (
+    <Page title="Welcome to Gestalt" isFullWidth>
+      <Box width="100%">
+        <Flex direction="column">
+          {/* Hero */}
+          <HomePageSection>
+            <Box display="flex" alignItems="center" justifyContent="between">
```

Comment:
we clearly don't enable our lint rules for our own repo 😬 😂 

any reason why this isn't Flex?

Reply:
Yeah, I need both the minWidth and display none on the second item inside this container, so I needed to use Box to avoid the Flex.Item being added

Reply:
Ah, makes sense. Probably worth leaving a comment about so it doesn't get changed if we start running our lint rules on this codebase, or in other future refactors